task5/src/main/java/com/clouway/adapter/http/LoginController.java

```diff
@@ -38,17 +41,17 @@ protected void doPost(HttpServletRequest request, HttpServletResponse response)
   }
 
   private void authorize(String email, String password, HttpServletResponse response) throws IOException, ServletException {
-    User user = userRepository.findByEmail(email);
-    if (user != null) {
-      if (user.getPassword().equals(password)) {
-        Cookie cookie = new Cookie("sessionId", email);
+    boolean authorize=userRepository.authorize(email,password);
+    if (authorize) {
```

Comment:
My mistake. It is `authentication`. Authentication is used when verifying username and password.